content/40.resource-availability.md

```diff
@@ -1,8 +1,24 @@
-## Data and Resource Availability
+## Resource Availability
 
-This manuscript was written [openly on GitHub](https://github.com/greenelab/iscb-diversity-manuscript) using Manubot [@doi:10.1371/journal.pcbi.1007128].
-The Manubot HTML version is available under a Creative Commons Attribution (CC BY 4.0) License at <https://greenelab.github.io/iscb-diversity-manuscript/>.
-Our analysis of authors and ISCB-associated honorees is available under CC BY 4.0 at <https://github.com/greenelab/iscb-diversity>, with source code also distributed under a BSD 3-Clause License.
-Rendered Python and R notebooks from this repository are browsable at <https://greenelab.github.io/iscb-diversity/>.
-Our analysis of PubMed, PubMed Central, and author names relies on the Python pubmedpy package, developed as part of this project and available under a Blue Oak Model License 1.0 at <https://github.com/dhimmel/pubmedpy> and on [PyPI](https://pypi.org/project/pubmedpy/).
+### Lead Contact
+Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Casey Greene (greenescientist@gmail.com).
```

Comment:
```suggestion
Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Casey Greene (casey.s.greene@cuanschutz.edu).
```
To comply with public records / FOIA requirements as a state employee I now use Anschutz for work-related emails 😁